tests/storage/hostdisk.go

```diff
@@ -44,3 +50,19 @@ func CreateDiskOnHost(diskPath string) *k8sv1.Pod {
 
 	return pod
 }
+
+func RemoveHostDiskImage(diskPath string, nodeName string) {
```

Comment:
I find the asymmetry between  `CreateDiskOnHost` and `RemoveHostDiskImage` almost painful.

Would you consider having a follow-up PR that makes them more similar e.g.
```
CreateHostDisk() (path, nodeName, error);
RemoveHostDisk(path, nodeName string) error;
```